src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -85,8 +85,8 @@ def _get_draft_version_fallback(strategy: str, sphinx_config: Dict[str, Any]):
     return '[UNRELEASED DRAFT]'
 
 
-def _nodes_from_rst(state: statemachine.State, rst_source: str) -> List[nodes.Node]:
-    """Turn an RST string into a list of nodes that can be used in the document."""
+def _nodes_from_rst(state: statemachine.State, rst_source: str) -> List[nodes.Node]:  # pylint: disable=line-too-long
```

Comment:
The proper fix is to make it fit the restrictions instead of ignoring the rule:
```suggestion
def _nodes_from_rst(
    state: statemachine.State,
    rst_source: str,
) -> List[nodes.Node]:
```